facia-press/app/frontpress/FapiFrontPress.scala

```diff
@@ -124,15 +124,8 @@ trait FapiFrontPress extends QueryDefaults with Logging with ExecutionContexts {
   }
 
   private def getBackfill(collection: Collection): Response[List[PressedContent]] = {
-    collection
-      .collectionConfig
-      .apiQuery
-      .map { query =>
-        FAPI.backfill(query, collection, searchApiQuery, itemApiQuery)
-      }
-      .getOrElse {
-        Response.Right(Nil)
-      }.map(_.map(PressedContent.make))
+    FAPI.backfillFromConfig(collection, searchApiQuery, itemApiQuery)
```

Comment:
why does `backfillFromConfig` take a `Collection`, and not the config in the collection? Should it be called `backfillFromCollection`?